gui/mozregui/global_prefs.py

```diff
@@ -16,6 +16,7 @@ def get_prefs():
     options = dict()
     options['persist'] = settings['persist']
     options['http_timeout'] = float(settings['http-timeout'])
+    options['persist-size-limit'] = settings['persist-size-limit']
```

Comment:
This should be:

```
options['persist_size_limit'] = float(settings['persist-size-limit'])
```
(note the use of underscores instead of dashes). We use dashes in the conf file, but underscores elsewhere.

Also note that we need to convert it to float.